Extensions/BBText/JsExtension.js

```diff
@@ -0,0 +1,511 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+
+const MultiStyleText = require('./dist/pixi-multistyle-text.umd');
+console.log(MultiStyleText);
+module.exports = {
+  createExtension: function(_, gd) {
+    const extension = new gd.PlatformExtension();
+    extension
+      .setExtensionInformation(
+        'BBText',
+        'BBTextObject',
+        _('Display a BBText object on the scene.'),
+        'Todor Imreorov',
+        'Open source (MIT License)'
+      )
+      .setExtensionHelpPath('/objects/BBTextObject');
```

Comment:
For consistency, use the style of other objects urls: http://wiki.compilgames.net/doku.php/gdevelop5/objects
No "Object" at the end, use underscore for separating word, everything lowercased